recipes/seqlike/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "seqlike" %}
+{% set version = "1.1.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/seqlike-{{ version }}.tar.gz
+  sha256: 9bf9f9af1b6ffef2d322b285424f4b89c6622aef1ccd9b88def4dac45d8107b8
+
+build:
+  number: 0
+  noarch: python
+  script:
+    {% set reqfile = "requirements.txt" %}
+    {% set reqsource = name ~ ".egg-info/requires.txt" %}
+    # IF {{ reqfile }} is NOT PRESENT, COPY {{ reqfile }} from {{ reqsource }}
+    - if [ ! -f {{ reqfile }} ] && [ -f {{ reqsource }} ]; then cp {{ reqsource }} {{ reqfile }}; fi  # [not win]
+    - copy {{ name }}.egg-info\requires.txt {{ reqfile }}  # [win] 
```

Comment:
Is this needed to solve the Windows error?

Reply:
Checked in the source-code. The `setup.py:install_requires` is hard-coded. So, that snippet is not required.

Reply:
Do you know of any situation where this might be required? Because this is generally not used in conda-forge recipes.